.travis.yml

```diff
@@ -6,6 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache
     - downloads
+    - vendor
```

Comment:
Are you sure that's a good idea? Don't know which side effects this might have, and I also can't see a big speed improvement.

Also, after some googling, I only found solutions where the composer cache is cached (what we already do) and not the `vendor` folder. Maybe that's also the reason there is no big boost here.